korea_news_crawler/sportcrawler.py

```diff
@@ -72,35 +81,30 @@ def make_sport_page_url(self, url, start_year, last_year, start_month, last_mont
 
             for month in range(target_start_month, target_last_month + 1):
                 for day in range(1, calendar.monthrange(year, month)[1] + 1):
-                    url = url
+                    url = input_url
                     if len(str(month)) == 1:
                         month = "0" + str(month)
                     if len(str(day)) == 1:
                         day = "0" + str(day)
                     url = f'{url}{year}{month}{day}'
                     # page 날짜 정보만 있고 page 정보가 없는 url 임시 저장
-                    final_url = url
+                    final_url = copy.deepcopy(url)
 
                     # TotalPage 확인
                     total_page = self.get_total_page(url)
```

Comment:
```suggestion
                    total_page = self.get_total_page(final_url)
```

이렇게 하면 url 변수 scope 문제도 없어져 deepcopy 할 필요도 없지 않을까요 ?

Reply:
아 죄송합니다 확인이 늦었네요.  위 내용을 그대로 local에 적용시켜 테스트해본 결과 크롤링이 잘 되었으니 제안해주신 내용으로 commit하는 것이 좋아보입니다.